scripts/flash/dispute-contribute.js

```diff
@@ -45,76 +44,96 @@ function disputeContribute(augur, args, auth, callback) {
-              displayTime("Current timestamp", timeResult.timestamp);
-              displayTime("Fee Window end time", feeWindowStartTime);
-              displayTime("Set Time to", setTime);
-              setTimestamp(augur, setTime, timeResult.timeAddress, auth, function (err) {
+              getTime(augur, auth, function (err, timeResult) {
                 if (err) {
                   console.log(chalk.red(err));
                   return callback(err);
                 }
-                console.log(chalk.yellow("sending amount REP"), chalk.yellow(attoREP), chalk.yellow(amount));
-                augur.api.FeeWindow.isActive(feeWindowPayload, function (err, result) {
+
+                var setTime = parseInt(feeWindowStartTime, 10) + day;
+                displayTime("Current timestamp", timeResult.timestamp);
+                displayTime("Fee Window end time", feeWindowStartTime);
+                displayTime("Set Time to", setTime);
+                setTimestamp(augur, setTime, timeResult.timeAddress, auth, function (err) {
                   if (err) {
                     console.log(chalk.red(err));
                     return callback(err);
                   }
-                  console.log(chalk.green.dim("Few Window is active"), chalk.green(result));
-                  if (result) {
-                    doMarketContribute(augur, marketId, attoREP, payoutNumerators, invalid, auth, function (err) {
-                      if (err) {
-                        return callback("Market contribute Failed");
-                      }
-                      console.log(chalk.green("Market contribute Done"));
-                      callback(null);
-                    });
-                  } else {
-                    console.log(chalk.red("Fee Window isn't active"));
-                    callback("Fee Window isn't active");
-                  }
+
+                  console.log(chalk.yellow("sending amount REP"), chalk.yellow(attoREP), chalk.yellow(disputeAmount));
+                  augur.api.FeeWindow.isActive(feeWindowPayload, function (err, result) {
+                    if (err) {
+                      console.log(chalk.red(err));
+                      return callback(err);
+                    }
+
+                    console.log(chalk.green.dim("Few Window is active"), chalk.green(result));
+                    if (result) {
+                      doMarketContribute(augur, marketId, attoREP, payoutNumerators, invalid, auth, function (err) {
+                        if (err) {
+                          return callback("Market contribute Failed");
+                        }
+
+                        console.log(chalk.green("Market contribute Done"));
+                        callback(null);
+                      });
+                    } else {
+                      console.log(chalk.red("Fee Window isn't active"));
+                      callback("Fee Window isn't active");
```

Comment:
This should also probably be `return callback`.

Reply:
just for convention, I agree. will fix